### DISCUSSION TITLE
Question about the submission of code& technical paper.

### DISCUSSION THREAD

**Discussion by AlvinAi96** [PaddlePaddle/PaddleSpatial#291] [BaiduKDDCup2022] [answered]
Hi, Host,

Do we need to submit a report? We just saw the website stated that : 
“
_July 19, Winner notification. Private notifications and instructions about the code& technical paper are sent to the awardees.
July 21, Code and the technical paper submission deadline for the awardees._
”
However, we haven't received any private notification about it until now. Does Paddle ranking is not included in the winner notification? 

By the way, this discussion [Question About Certifications #288](https://github.com/PaddlePaddle/PaddleSpatial/discussions/288) told us that we could get a certificate at least. How to get this certificate? Are any instructions for us to write the technical paper or code? 

Look forward to your reply.

Alvin Ai

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

1) we have sent out the notifications to the award candidate teams for both regular track and paddle track. If you do not receive the notification email, please send your team name to us (kddcup2022@baidu.com), and we will check it.

2) If you want to obtain a certificate, please also send a request to us before July 21.  The organizer committee will provide a certificate of successful participation to your team if you can submit a report and the code no matter your rank.


Thanks a lot.

**Reply by AlvinAi96** [reply]
Thanks for informing.
